railties/lib/rails/generators/app_base.rb

```diff
@@ -467,6 +467,8 @@ def hotwire_gemfile_entry
       end
 
       def using_node?
+	return if options[:javascript] == "bun"
```

Comment:
this is because bun can be used alongside with `tailwind` or `sass`. So when adding `tailwind` with `bun`, we set `using_node?` to false